ml-agents/mlagents/trainers/sac/optimizer_torch.py

```diff
@@ -202,10 +202,22 @@ def sac_q_loss(
         return q1_loss, q2_loss
 
     def soft_update(self, source: nn.Module, target: nn.Module, tau: float) -> None:
-        for source_param, target_param in zip(source.parameters(), target.parameters()):
-            target_param.data.copy_(
-                target_param.data * (1.0 - tau) + source_param.data * tau
-            )
+        """
+        Performs an in-place polyak update of the target module based on the source,
```

Comment:
I am actually not sure this is Polyak update. I would say `Exponential moving average update`

Reply:
Mention that a tau = 1 means replace the target network weights with the source.

Reply:
OpenAI and Stable Baselines refer to it as polyak: 
![image](https://user-images.githubusercontent.com/5085265/93921424-69159380-fcc5-11ea-8eea-010305d5128c.png)

Added the comment about tau